java/java.hints/src/org/netbeans/modules/java/hints/jdk/CheckRegexTopComponent.java

```diff
@@ -0,0 +1,249 @@
+                    .addComponent(RegexScrollPane, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                    .addComponent(ExampleScrollPane, javax.swing.GroupLayout.DEFAULT_SIZE, 280, Short.MAX_VALUE))
+                .addContainerGap(355, Short.MAX_VALUE))
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addContainerGap()
+                .addComponent(RegexLabel)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
+                .addComponent(RegexScrollPane, javax.swing.GroupLayout.PREFERRED_SIZE, 42, javax.swing.GroupLayout.PREFERRED_SIZE)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
+                .addComponent(ExampleLabel)
+                .addGap(15, 15, 15)
+                .addComponent(ExampleScrollPane, javax.swing.GroupLayout.PREFERRED_SIZE, 41, javax.swing.GroupLayout.PREFERRED_SIZE)
+                .addContainerGap(15, Short.MAX_VALUE))
+        );
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void ExampleTextAreaKeyReleased(java.awt.event.KeyEvent evt) {//GEN-FIRST:event_ExampleTextAreaKeyReleased
+        matchPattern();
+    }//GEN-LAST:event_ExampleTextAreaKeyReleased
+
+    private void RegexTextAreaKeyReleased(java.awt.event.KeyEvent evt) {//GEN-FIRST:event_RegexTextAreaKeyReleased
+        matchPattern();
+    }//GEN-LAST:event_RegexTextAreaKeyReleased
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JLabel ExampleLabel;
+    private javax.swing.JScrollPane ExampleScrollPane;
+    private javax.swing.JTextArea ExampleTextArea;
+    private javax.swing.JLabel RegexLabel;
+    private javax.swing.JScrollPane RegexScrollPane;
+    private javax.swing.JTextArea RegexTextArea;
+    // End of variables declaration//GEN-END:variables
+    @Override
+    public void componentOpened() {
+        // TODO add custom code on component opening
+    }
+
+    @Override
+    public void componentClosed() {
+        // TODO add custom code on component closing
+    }
+
+    void writeProperties(java.util.Properties p) {
+        // better to version settings since initial version as advocated at
+        // http://wiki.apidesign.org/wiki/PropertyFiles
+        p.setProperty("version", "1.0");    // NOI18N
+        // TODO store your settings
+    }
+
+    void readProperties(java.util.Properties p) {
+        String version = p.getProperty("version");  // NOI18N
+        // TODO read your settings according to their version
+    }
+
+    public static synchronized CheckRegexTopComponent getDefault() {
+        if (instance == null) {
+            instance = new CheckRegexTopComponent();
+        }
+        Mode outputMode = WindowManager.getDefault().findMode("output");
```

Comment:
If you don't expect to be transrate the string, please add `// NOI18N`
below 
- "CheckRegexTopComponent"
-  "Cannot find " + "CheckRegexTopComponent" + " component. It will not be located properly in the window system."
-  "There seem to be multiple components with the '" + "CheckRegexTopComponent"
                + "' ID. That is a potential source of errors and unexpected behavior."
 as well